src/d3a/models/strategy/load_hours.py

```diff
@@ -72,7 +72,9 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None, daily_budget=
     def active_markets(self):
         markets = []
         for market in self.area.all_markets:
-            if self._allowed_operating_hours(market.time_slot):
+            if self._allowed_operating_hours(market.time_slot) and \
+                    self.area.config.start_date <= market.time_slot <= \
```

Comment:
Since it is used in 2 places, I wonder if it makes sense to refactor this to a simple util function named 'is_market_in_simulation_duration' or similar.